tracer/src/Datadog.Tracer.Native/iast/dataflow_aspects.cpp

```diff
@@ -48,19 +48,32 @@ namespace iast
     }
 
     //------------------------------------
+    VersionInfo currentVersion = GetVersionInfo(GetDatadogVersion());
 
     DataflowAspectClass::DataflowAspectClass(Dataflow* dataflow, const WSTRING& aspectsAssembly, const WSTRING& line)
     {
         this->_dataflow = dataflow;
-        //[AspectClassAttribute("mscorlib,netstandard,System.Private.CoreLib",PROPAGATION,"")] Hdiv.AST.Aspects.Aspects.System_StringAspect
         this->_aspectsAssembly = aspectsAssembly;
         this->_line = line;
         size_t offset = 0;
         auto pos0 = IndexOf(line, WStr("[AspectClass("), &offset);
         if (pos0 == std::string::npos) { return; }
         pos0 = offset;
         auto pos1 = IndexOf(line, WStr(")] "), &offset);
-        if (pos1 == std::string::npos) { return; }
+        if (pos1 == std::string::npos) 
+        {
+            //Check for version limitation
+            pos1 = IndexOf(line, WStr(");V"), &offset);
```

Comment:
Just to be clear, this is the only place `;` _can_ be in the string right? So this is definitely going to get our expected position

Reply:
Yes, that's it